remove_context.py

```
@@ -0,0 +1,38 @@
+import re
+
+directory = "src/main/java"
+pattern = re.compile(r'^(\s*(?://|\*|/\*)\s*(?:Clinical\s*)?)Context:\s*(.*)', re.IGNORECASE)
+
+count = 0
+for root, _, files in os.walk(directory):
+    for file in files:
+        if file.endswith(".java"):
+            filepath = os.path.join(root, file)
+            try:
+                with open(filepath, 'r') as f:
```

Ruff (0.15.12)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)